codalab/lib/codalab_manager.py

```diff
@@ -247,9 +248,13 @@ def bundle_store(self):
         depends on what the user has configured, but if no bundle store is configured manually then it defaults to a
         MultiDiskBundleStore.
         """
-        store_type = self.config.get('bundle_store', 'MultiDiskBundleStore')
+        store_type = self.config.get('bundle_store', 'MultiDiskBundleStoreWithBlobStorage')
```

Comment:
Not sure we should change this default...by default, people spinning up an instance will likely use the MultiDiskBundleStore.

Reply:
Ok, yeah, this is one abstraction I wasn't sure if we needed. Do you think we really need a separate `MultiDiskBundleStoreWithBlobStorage` class, or do you think it would be best to just modify the existing `MultiDiskBundleStore` to also check Blob Storage first?

Reply:
FWIW, we had abstractions in the past (SingleDisk and MultiDisk) and we just removed it because it was kind of annoying.